orttraining/orttraining/training_api/module.h

```diff
@@ -73,12 +74,11 @@ struct Module {
  public:
   // Initialize a module from an ORT inference session with loaded
   // training ONNX model and load parameters
-  Module(const std::string& train_model_path_or_bytes,
+  Module(const ModelIdentifiers& model_identifiers,
```

Comment:
can we add some comment about the laternatives when creating Module, e.g. either initialized from a model path or from a model data path.

Reply:
Done